src/tests/ftest/erasurecode/rebuild_disabled_single.yaml

```diff
@@ -15,7 +15,6 @@ setup:
 server_config:
   name: daos_server
   engines_per_host: 2
-  crt_timeout: 10
```

Comment:
Sorry, I meant you could remove the change to add this in your PR because it's already added now. So we need to keep this

Reply:
Done. Thanks for the comment.